accessibility.html

```diff
@@ -0,0 +1,283 @@
+                        if (request.status >= 200 && request.status < 400) {
+                            // We save the file in localStorage
+                            localStorage.firaSansWeb = request.responseText;
+
+                            // ... and load the font
+                            addFont();
+                        }
+                    };
+
+                    request.send();
+                }
+            } catch(ex) {
+                // maybe load the font synchronously for woff-capable browsers
+                // to avoid blinking on every request when localStorage is not available
+            }
+        }());
+      </script>
+    </head>
+    <body class="gray">
+      <header role="banner">
+        <section class="nav"></section>
+      </header>
+      <div class="content" role="main">
+        <div class="box">
+          <section class="cb teaser">
+            <aside role="complementary">
+              <br />
+              <br />
+              <br />
+              <br />
+              <nav role="navigation">
+                <a href="index.html">0. Intro &amp; Contribute</a><br />
+                <a href="index.html">1. Preq &amp; Setup</a><br />
+                <a href="index.html">1. Files, what &amp; why?</a><br />
+                <a href="index.html">2. Build your Website</a><br />
+                <a href="colour-fonts.html">3. Colours and Fonts</a><br />
+                <a href="images.html">4. Images</a><br />
+                <a href="index.html">5. Run Deployment</a>
+              </nav>
+            </aside>
+            <article role="article">
+              <h1>Accessibility</h1>
+              <p>
+                Making sure that everything we do at Hoodie, is accessible to everyone.
+              </p>
+              <ul>
+                <li><a href="#introduction">Introduction</a></li>
+                <li><a href="#factors">Factors affecting experiences with the web</a></li>
+                <li><a href="#guidelines">Accessibility guidelines</a></li>
+                <li><a href="#example">An example of making Hoodie accessible</a></li>
+                <li><a href="#tools">Accessibility testing</a></li>
+              </ul>
+            </article>
+          </section>
+
+          <section class="cb">
+            <article id="introduction" role="article">
+              <h2>Introduction</h2>
+              <p>
+                In 1997, the W3 launched the "International Program Office for Web Accessibility Initiative".
+                In the <a href="http://www.w3.org/Press/IPO-announce">press release</a>, Tim Berners-Lee said the following:
+                "The power of the Web is in its universality. Access by everyone regardless of disability is an essential aspect".
+              </p>
+              <p>
+                One of the first things you see when you go to the Hoodie website is the heading "Hoodie is for you".
```

Comment:
"Hoodie Website" is showing up in brackets when I run this local, but I don't see them here, so not sure what is happening.